src/core.js

```diff
@@ -755,11 +755,30 @@ define([
             }
 
             if (window.analytics) {
-                window.analytics.usage(
-                    "Question Reference",
-                    "Drag and Drop",
-                    _this.data.core.currentlyEditedProperty
-                );
+                var targetType;
+                switch (target[0].id) {
+                    case 'property-relevantAttr':
+                        targetType = "Display";
+                        break;
+                    case 'property-constraintAttr':
+                        targetType = "Validation";
+                        break;
+                    case 'property-calculateAttr':
+                        targetType = "Calculation";
+                        break;
+                    default:
+                        targetType = "Expression Editor";
+                        break;
+                }
+                if (path.contains('casedb')) {
```

Comment:
Looks like this will catch any path containing `casedb`, including something like `/data/not_in_my_casedb`. Seems like we might want to swap the branches here and do something like `path.startsWith("/data/")` to detect form references. Also should test this on reference to the current question, which uses a relative path of `.` (I can't remember if the relative path is passed in here or not).